build.gradle.kts

```diff
@@ -5,29 +5,38 @@ plugins {
   idea
 }
 
+dependencyCheck {
+  suppressionFiles.add("reactive-suppressions.xml")
```

Comment:
needed as had to pin to old version of h2 and also false positives on some netty classes